### PR TITLE
Add cyclonedx command

### PR DESCRIPTION
**Pull request by jdalton**
Add `cyclonedx` command and update deps.

### CODE REVIEW

File: lib/commands/cyclonedx/index.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      },
+      stdout: 'inherit'
+    })`cdxgen ${argvToArray(yargv)}`
+
+    if (cleanupPackageLock) {
+      try {
+        await fs.unlink('./package-lock.json')
+      } catch {}
+    }
+    console.log(chalk.cyanBright(`${yargv.output} created!`))
```
Comment (charliegerard) [inline lib/commands/cyclonedx/index.js:206]: When running the command `cyclonedx --help`, this message is printed even though the file has not been created.

**Review by charliegerard — APPROVED** [review]
Just left a small comment, otherwise looks good!

File: lib/commands/cyclonedx/index.js (hunk 1 above)
Comment (jdalton) [reply lib/commands/cyclonedx/index.js:206]: Oh that's a good catch. Thank you!

Comment (jdalton) [reply lib/commands/cyclonedx/index.js]: Updated to only log if the file exists.

**Pull request merged**